deploy/crd/cluster.open-cluster-management.io_clustercurators.yaml

```diff
@@ -0,0 +1,245 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: clustercurators.cluster.open-cluster-management.io
+spec:
+  group: cluster.open-cluster-management.io
+  names:
+    kind: ClusterCurator
+    listKind: ClusterCuratorList
+    plural: clustercurators
+    singular: clustercurator
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      description: ClusterCurator is the Schema for the clustercurators API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ClusterCuratorSpec defines the desired state of ClusterCurator
+          properties:
+            curatorJob:
+              description: Kubernetes job resource created for curation of a cluster
+              type: string
+            desiredCuration:
+              description: This is the desired curation flow
+              type: string
+            destroy:
+              description: During an destroy curation run these **Pre hook ONLY**
+              properties:
+                overrideJob:
+                  description: When provided, this is a Job specification
+                  type: object
+                posthook:
```

Comment:
pre hook only?

Reply:
Because things run inside the cluster repository, there is no real way to do post work as the namespace is gone.

We'll revisit when we're there, but I don't think it will be available